cross/rest-server/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = rest-server
+PKG_VERS = 0.11.0
+PKG_COMMIT_ID = 4741eec619a2b61d251ffbd0acce028620067579
```

Comment:
AFAICS this variable is not used
```suggestion
```

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/5550/commits/f85d69a462510d7a7d1ca70d1ae1e01209f2d426